WeakAuras/WeakAuras.lua

```diff
@@ -1607,14 +1629,18 @@ local function GetInstanceTypeAndSize()
   return "none", "none", nil, nil, nil
 end
 
+---@return string instanceType
+---@return nil
```

Comment:
I guess the cleaner implementation of InstanceType would be:
``` 
return (GetInstanceTypeAndSize())
```

Reply:
done